### ISSUE TITLE
Metadata for "xxx" does not exist, cannot launch

### ISSUE THREAD

**Issue by ommer15** [Heroic-Games-Launcher/HeroicGamesLauncher#1198]
### Describe the bug

when attempting to launch already installed games from the launcher it will attempt at launching, but apparently fail due to a "metadata error", claiming that the metadata file doesn't exist. after looking deeper into the `.config/legendary/metadata` folder i was able to locate the metadata file and everything looked okay to me

though i noticed this happened to me only with gta v and rocket league so far. to see if it would happen with another game i installed absolute drift (mac version) and it ran just fine with 0 metadata errors. and yes the metadata for absolute drift did exist there, obviously. i could only assume that this has something to do with different games running on their non matching platforms
by the way i checked all of my wine settings after settings them up with crossover and they all seem to be fine as well

### Add logs

**GTAV launch log:**

(09:45:04) INFO:    [Legendary]:       finished login
(09:45:04) INFO:    [Frontend]:        Called Login
(10:57:04) INFO:    [Legendary]:       Getting More details with Legendary info command... 
(10:57:11) INFO:    [Backend]:         launching Grand Theft Auto V 9d2d0eb64d5c44529cece33fe2a46482
(10:57:11) INFO:    [Legendary]:       Getting More details with Legendary info command... 
(10:57:12) INFO:    [Legendary]:       Launch Command: 
(10:57:27) ERROR:   [Backend]:         Error - Command failed: legendary launch 9d2d0eb64d5c44529cece33fe2a46482    
[cli] INFO: Logging in...
[Core] INFO: Trying to re-use existing login session...
[cli] INFO: Checking for updates...
[cli] CRITICAL: Metadata for "9d2d0eb64d5c44529cece33fe2a46482" does not exist, cannot launch!

**Absolute Drift launch log:** 

(11:40:37) WARNING: [Backend]:         Protocol already registered.
(11:40:38) INFO:    [Frontend]:        Refreshing Library
(11:40:38) INFO:    [Legendary]:       Refreshing library...
(11:40:38) INFO:    [Legendary]:       Refreshing Epic Games...
(11:40:38) INFO:    [Legendary]:       Refreshing Epic Games...
(11:40:44) INFO:    [Backend]:         checking for heroic updates
(11:40:47) INFO:    [Backend]:         launching Absolute Drift 19927295d6e3467887d4e830d8c85963
(11:40:49) INFO:    [Legendary]:       [cli] INFO: Logging in...
[Core] INFO: Trying to re-use existing login session...
(11:40:49) INFO:    [Legendary]:       Launch Command: 
(11:40:49) INFO:    [Legendary]:       [cli] INFO: Getting game list... (this may take a while)
(11:41:00) INFO:    [Legendary]:       Updating game list
(11:41:00) INFO:    [Legendary]:       Game List Updated
(11:41:26) INFO:    [Legendary]:       Checking for game updates
(11:41:26) INFO:    [Legendary]:       Found 0 game(s) to update
(11:41:26) INFO:    [Gog]:             Found 0 game(s) to update
(11:41:56) INFO:    [Backend]:         Log was written to /Users/omer/Library/Application Support/heroic/GamesConfig/19927295d6e3467887d4e830d8c85963-lastPlay.log

### Steps to reproduce

- open heroic
- launch any windows game
- navigate to settings -> log in case the game hasnt launched

### Expected behavior

the game(s) would launch normally

### System Information

- OS [macOS 12.3.1]: 
- Heroic Version [2.2.6]:

**Comment by flavioislima** [comment]
Can you provide the full log?
This seems like a Legendary error message since Heroic doesnt launch the game itself, but calls legendary.

**Comment by iamroot007** [comment]
Hi @wiryfuture @flavioislima I am also getting the same issue.Below is full error message.

```
Launch Command: /Applications/Heroic.app/Contents/Resources/app.asar.unpacked/build/bin/darwin/legendary launch 9d2d0eb64d5c44529cece33fe2a46482

System Info:

  Heroic Version: 2.3.3 Brook
  Legendary Version:  0.20.25 Our Benefactors
  OS: macOS KERNEL: 21.5.0 ARCH: x64
  CPU: Apple M1 @2.4 
  RAM: Total: 16 GiB Available: 11 GiB
  GRAPHICS: GPU0: Apple M1 VRAM: nullMB DRIVER:  
  
  

Game Settings: {
	"autoInstallDxvk": false,
	"autoInstallVkd3d": false,
	"maxSharpness": 2,
	"nvidiaPrime": false,
	"otherOptions": "",
	"savesPath": "",
	"showFps": false,
	"targetExe": "",
	"useGameMode": false,
	"wineCrossoverBottle": "Heroic",
	"winePrefix": "/Users/<user>/Games/Heroic/Prefixes/Default",
	"wineVersion": {
		"bin": "'/Applications/CrossOver.app/Contents/SharedSupport/CrossOver/bin/wine'",
		"name": "CrossOver - 21.2",
		"type": "crossover",
		"wineserver": "'/Applications/CrossOver.app/Contents/SharedSupport/CrossOver/bin/wineserver'",
		"wineboot": "'/Applications/CrossOver.app/Contents/SharedSupport/CrossOver/bin/wine' wineboot"
	},
	"useSteamRuntime": false
}

Error Log:
[cli] INFO: Logging in...
[Core] INFO: Trying to re-use existing login session...
[cli] INFO: Checking for updates...
[cli] CRITICAL: Metadata for "xxxxxx" does not exist, cannot launch!
```

**Comment by flavioislima** [comment]
To fix this we need to add support for platform selection when importing.
Looking at the code, we do not check the selected platform right now.

**Comment by flavioislima** [comment]
This was fixed on 2.5.0 since now the import will respect the selected platform on the modal.
